src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/Example2.java

```diff
@@ -0,0 +1,22 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="OverloadMethodsDeclarationOrder"/>
```

Comment:
example with same config look weird in comparison with others.

lets make example1
```
class Example1 {
  void same(int i) {}
  // comments between overloaded methods are allowed.
  void same(String s) {}
  void same(String s, int i) {}
  void same(int i, String s) {}
  void notSame() {}
  interface notSame{}

  void otherSame(String s) {}
  void foo() {}
  // violation below
  void otherSame(String s, int i) {}
}
```

and remove Example2.


change description to `Example of correct and incorrect grouping of overloaded methods` to

Reply:
Done